modules/statics/src/coins.ts

```diff
@@ -699,6 +700,7 @@ export const coins = CoinMap.fromCoins([
   erc20('stbu', 'Stobox Token', 18, '0x212dd60d4bf0da8372fe8116474602d429e5735f', UnderlyingAsset.STBU),
   erc20('stc', 'Student Coin', 2, '0xb8b7791b1a445fb1e202683a0a329504772e0e52', UnderlyingAsset.STC),
   erc20('stcv2', 'Student Coin V2', 18, '0x15b543e986b8c34074dfc9901136d9355a537e7e', UnderlyingAsset.STCV2),
+  erc20('seth-h', 'Staked Aave', 18, '0x65077fa7df8e38e135bd4052ac243f603729892d', UnderlyingAsset.SETHH),
```

Comment:
Then you can bracket notation it here and keep the one-to-one token name you are looking for.